client/src/pages/guide/english/security/ddos-denial-of-service-attacks/index.md

```diff
@@ -39,6 +39,11 @@ Protocol attacks include SYN floods, Ping of Death, fragmented packet attacks, S
 
 * **Ping of Death** − The attacker sends malformed or oversized packets using a simple ping command. IP allows sending 65,535 bytes packets but sending a ping packet larger than 65,535 bytes violates the Internet Protocol and could cause memory overflow on the target system and finally crash the system. To avoid Ping of Death attacks and its variants, many sites block ICMP ping messages altogether at their firewalls.
 
+* **Industrial IOT Attacks based on DDOS** - The attacker analyses the communication network and sniffs the packets using packet capturing tools. After deep packet inspection, the attacker checks the command frame which can stop the data transmission and crafts the packets to disrupt the Industrial control systems. Various protocols like IEEE C37.118.2, Modbus, DNP3, etc are vulnerable.
+For eg. * [Blackenergy 3 Malware] (https://en.wikipedia.org/wiki/December_2015_Ukraine_power_grid_cyberattack)
```

Comment:
the link is not in proper format, likely just the extra space broke it. Plus, "For eg. *" doesn't make any sense, I suggest "Example: ".